spec/controllers/concerns/two_factor_authenticatable_methods_spec.rb

```diff
@@ -189,6 +193,10 @@
           multi_factor_auth_method: TwoFactorAuthenticatable::AuthMethod::SMS,
           enabled_mfa_methods_count: 1,
           new_device: true,
+          mfa_attempts: {
+            attempts: 1,
+            auth_method: 'sms',
+          },
```

Comment:
I think now that we're only tracking attempts for the current method, and because `auth_method` is a redundant property, it'd be nice if we logged the numeric value of attempts instead.

```suggestion
          attempts: 1,
```

Reply:
Revised expected analytics event with [e6ab87d](https://github.com/18F/identity-idp/pull/11293/commits/e6ab87da72b2025b6215107a704880cbb8a144cb)